test/e2e/providers/exec/docker.go

```diff
@@ -17,9 +17,12 @@ package exec
 import (
 	"fmt"
 	"io"
+	"log"
 	"os/exec"
 	"strings"
 	"sync"
+
+	"github.com/google/shlex"
```

Comment:
This project was archived and is no longer maintained. Do we really want to add it as a new dependency?